### PR TITLE
Mobile improvements

### PR DESCRIPTION
**Pull request by LinqLover**
Several CSS changes that avoid horizontal overflows or truncated texts on mobile.

### CODE REVIEW

File: evap/static/scss/components/_buttons.scss [hunk 1]
```diff
@@ -158,7 +158,7 @@ a:not([href]):not(.disabled) {
     .button-open-notebook {
         position: fixed;
         bottom: $footer-height + 20px;
-        z-index: $zindex-fixed;
+        z-index: $zindex-sticky - 1;
```
Comment (richardebeling) [inline evap/static/scss/components/_buttons.scss:161]: What is the rationale here: Why sticky instead of fixed, why -1? Can we add a comment?

**Review by richardebeling — COMMENTED** [review]
Thanks :)

File: evap/static/scss/components/_buttons.scss (hunk 1 above)
Comment (LinqLover) [reply evap/static/scss/components/_buttons.scss:161]:
```suggestion
        z-index: $zindex-sticky - 1;  /* be behind sticky navbar menus */
```

Comment (richardebeling) [reply evap/static/scss/components/_buttons.scss:161]: Yup, sounds good :+1:

**Review by richardebeling — APPROVED** [review]
Code looks good to me :)

File: evap/staff/templates/staff_semester_view.html [hunk 2]
```diff
@@ -259,7 +259,7 @@ <h3>
                         </div>
                     </div>
                 </div>
-                <form id="evaluation_operation_form" method="GET" action="{% url 'staff:evaluation_operation' semester.pk %}">
+                <form id="evaluation_operation_form" class="table-responsive" method="GET" action="{% url 'staff:evaluation_operation' semester.pk %}">
```
Comment (niklasmohrin) [inline evap/staff/templates/staff_semester_view.html:262]: Not sure we want the `table-*` class on the form here, shouldn't we instead apply it to the nested tables?

File: evap/static/scss/components/_staff-index.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+        .card {
+            display: inline-block;
+            width: 100%;
+        }
+    }
+
+    .card {
+        margin-bottom: 1rem;
+    }
```
[the remark this answers is not in the document]
Comment (LinqLover) [reply evap/static/scss/components/_staff-index.scss:16]: Sure, can commit this if it actually works :)

File: evap/staff/templates/staff_semester_view.html (hunk 2 above)
Comment (LinqLover) [reply evap/staff/templates/staff_semester_view.html:262]: > Make any table responsive across all viewports by wrapping a .table with .table-responsive.

https://getbootstrap.com/docs/5.3/content/tables/#responsive-tables

So the `table-responsive` is actually applied to the (or any) parent element of the table. :)

File: evap/static/scss/components/_distribution-bar.scss [hunk 4]
```diff
@@ -62,6 +62,10 @@
         margin: 0 0.5rem;
     }
 
+    &:not(:empty) {
+        min-width: calc(3ch + 2*3px);
```
[the remark this answers is not in the document]
Comment (LinqLover) [reply evap/static/scss/components/_distribution-bar.scss:66]: It's a while ago, but I guess for the following reason:

```suggestion
        min-width: calc(
            3ch /* up to 3 characters (e.g., 42%). This constraint is not relevant for 4 characters (100%) when there will be a single bar only. */
            + 2*3px /* left and right border */
        );
```

Feel free to commit this suggestion if we want to have the explanation in the sources :)

Comment (janno42) [inline evap/static/scss/components/_distribution-bar.scss:66]:
```suggestion
        min-width: calc(3ch + 2*3px); // 3 characters + left and right border
```

File: evap/static/scss/components/_buttons.scss [hunk 5]
```diff
@@ -158,7 +158,7 @@ a:not([href]):not(.disabled) {
     .button-open-notebook {
         position: fixed;
         bottom: $footer-height + 20px;
-        z-index: $zindex-fixed;
+        z-index: $zindex-sticky - 1;  /* be behind sticky navbar menus */
```
[the remark this answers is not in the document]
Comment (richardebeling) [reply evap/static/scss/components/_buttons.scss:161]: I tried getting the current z-index (on current main) of the button to cause some broken rendering but couldn't get it to do that :/ I'm especially unable to get the button to be in some place where a navbar dropdown is rendered. @LinqLover could you look into this again?

**Review by richardebeling — APPROVED** [review]

**Comment by LinqLover** [comment]
Sorry for ghosting you, this had somehow escaped my attention in my cluttered inbox! Thanks for merging!

**Pull request merged**